src/systems/systems.component.tsx

```diff
@@ -164,7 +164,7 @@ const columns: MRT_ColumnDef<System>[] = [
   },
 ];
 
-const MIN_SUBSYSTEMS_WIDTH = '320px';
+const MIN_SUBSYSTEMS_WIDTH = '480px';
```

Comment:
I think I would prefer to just have the pagination below the total, this adds a lot of empty space and is worse for small screens without changing the breakpoints further.